src/Files.Uwp/Dialogs/DynamicDialog.xaml.cs

```diff
@@ -20,6 +21,18 @@ public DynamicDialogResult DynamicResult
             get => ViewModel.DynamicResult;
         }
 
+        public new async Task<ContentDialogResult> ShowAsync() => await SetContentDialogRoot(this).ShowAsync();
+
+        // WINUI3
```

Comment:
What does this comment indicate?

Reply:
Nothing much, it only marks that the following method was added automatically when running the converter from uwp to winui3.

Reply:
Interesting, guess it does that to let you know it was behind the change.